python/tutorials/09-persistent-fp8-matmul.py

```diff
@@ -0,0 +1,409 @@
+import argparse
+import time
+
+import numpy as np
+import torch
+import triton
+import triton.language as tl
+import triton.profiler as proton
+
+from triton._C.libtriton import nvidia
+
+cublas_workspace = torch.empty(32 * 1024 * 1024, device="cuda", dtype=torch.uint8)
+cublas = nvidia.cublas.CublasLt(cublas_workspace)
+
```

Comment:
should we have some checks to make sure this is hopper? I believe we run the tutorials for our documentation and that would crash it. (I could be wrong, @Jokeren would know the doc part better). But in anycase better to not crash on other plarform

Reply:
The doc workflow is indeed run on A100

Reply:
good point! I've merged the PR since the message on slack was posted, but we should make another PR to repair the docs workflow.